frontend/src/components/Envoy/EnvoyDetails.tsx

```diff
@@ -23,6 +23,7 @@ import { SummaryTableBuilder } from './tables/BaseTable';
 import { Namespace } from 'types/Namespace';
 import { kialiStyle } from 'styles/StyleUtils';
 import AceEditor from 'react-ace';
+import 'ace-builds/src-noconflict/theme-twilight';
```

Comment:
Why theme-twilight is imported here but not theme-eclipse? Is this import needed here or we have to add theme-eclipse too?

Reply:
When I've tried the theme was not working, so I've added the import statement. 
I thought the eclipse was included as a default.